pkg/build/strategies/onbuild/onbuild_test.go

```diff
@@ -7,13 +7,17 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/openshift/source-to-image/pkg/build"
+	"github.com/openshift/source-to-image/pkg/build/ignore/docker"	
 	"github.com/docker/docker/builder/parser"
 
 	"github.com/openshift/source-to-image/pkg/api"
 	"github.com/openshift/source-to-image/pkg/test"
 )
 
-type fakeSourceHandler struct{}
+type fakeSourceHandler struct{
```

Comment:
what's special about ignorer that fakeSourceHandler needs Ignorer populated, but doesn't need Preparer and Downloader populated, for this test?